### PR TITLE
fix ErrorCode:AuthorizationPermissionMismatch issue

### PR DESCRIPTION
**Pull request by LiliDeng**
current code throws below exception when invoke get_blob_properties, probably because they are v1 type, not sure the RCA, but updated code works.
```"HttpResponseError: Operation returned an invalid status 'This request is not authorized to perform this operation using this permission.' ErrorCode:AuthorizationPermissionMismatch"```

### CODE REVIEW

File: lisa/sut_orchestrator/azure/platform_.py [hunk 1]
```diff
@@ -2136,27 +2147,39 @@ def _enable_ssh_on_windows(self, node: Node) -> None:
         log.dump_json(logging.DEBUG, result)
 
     def _get_vhd_os_disk_size(self, blob_url: str) -> int:
-        vhd_blob = BlobClient.from_blob_url(
-            blob_url=blob_url,
+        matched = STORAGE_CONTAINER_BLOB_PATTERN.match(blob_url)
```
Comment (squirrelsc) [inline lisa/sut_orchestrator/azure/platform_.py:2150]: The from blob url should wrap this logic already. When does not it work?

Comment (LiliDeng) [reply lisa/sut_orchestrator/azure/platform_.py:2150]: any feedback?

Comment (squirrelsc) [reply lisa/sut_orchestrator/azure/platform_.py:2150]: Can you paste the error again? I want to double check if from_blob_url work with this scenario or not.

Comment (LiliDeng) [reply lisa/sut_orchestrator/azure/platform_.py:2150]: deployment failed. HttpResponseError: Operation returned an invalid status 'This request is not authorized to perform this operation using this permission.'
ErrorCode:AuthorizationPermissionMismatch

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**